client/src/components/previews/OrganizationPreview.tsx

```diff
@@ -249,87 +256,6 @@ const OrganizationPreview = ({ className, uuid }: OrganizationPreviewProps) => {
         )}
       </div>
 
-      <h4>APP-06 symbology</h4>
-      <div className="preview-section">
-        <DictionaryField
-          wrappedComponent={PreviewField}
-          dictProps={Settings.fields.organization.app6context}
-          value={
-            (parentContext && (
-              <em>
-                {
-                  Settings.fields.organization.app6context.choices[
-                    parentContext
```

Comment:
parentContext, parentStandardIdentity and parentSymbolSet (defined above) are now unused and should be removed.